collect_app/google-services.json

```diff
@@ -34,7 +34,7 @@
           "other_platform_oauth_client": []
         },
         "ads_service": {
-          "status": 1
+          "status": 2
```

Comment:
are changes with `google-services.json` necessary?

Reply:
Reverted the commit `Rebasing fork` that did these changes.

Thanks @huangyz0918